scripts/cot/botania/flowers/functional/amuileria_kaerunea_late.zs

```diff
@@ -0,0 +1,98 @@
+#loader crafttweaker reloadable
+
+import crafttweaker.damage.IDamageSource;
+import crafttweaker.data.IData;
+import crafttweaker.item.IItemStack;
+import crafttweaker.util.Math;
+import crafttweaker.world.IBlockPos;
+import crafttweaker.world.IWorld;
+import mods.contenttweaker.Color;
+import mods.contenttweaker.VanillaFactory;
+import mods.randomtweaker.cote.BlockAdded;
+import mods.randomtweaker.cote.ISubTileEntity;
+import mods.randomtweaker.cote.ISubTileEntityFunctional;
+import mods.randomtweaker.cote.SubTileEntityInGame;
+import native.net.minecraft.util.EnumParticleTypes;
+import native.net.minecraft.world.WorldServer;
+
+static recipesLigthningFlower as IItemStack[string] = {
+} as IItemStack[string];
+
+static manaCostPerLightning as int = 1000;
+
+val amuileria_kaerunea = <cotSubTile:amuileria_kaerunea>;
+
+amuileria_kaerunea.onUpdate = function(subtile, world, pos) {
+    if(!world.worldInfo.isThundering()) return;
+    if(world.isRemote()) return;
+    if(world.time%100 == 17) charge(world, pos, subtile); 
+    return;
+};
+
+
+events.onEntityItemDeath(function (e as mods.zenutils.event.EntityItemDeathEvent) {
+    val world = e.item.world;
+    if (world.isRemote()) return;
+    val damageSource = e.damageSource;
+    if(isNull(damageSource)) return;
+    if(damageSource.damageType != 'lightningBolt') return;
+    val item = e.item;
+    if(isNull(item)) return;
+    //if(isNull(item.nbt.amuileria)) return;
```

Comment:
Please remove unused comments or describe how they could be useful if they supposed to be uncommented.